src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -21706,6 +21706,484 @@ void M(string? s)
                 );
         }
 
+        // TODO2
```

Comment:
Please address or file a bug.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/37238](https://github.com/dotnet/roslyn/issues/37238)

---
In reply to: [303557804](https://github.com/dotnet/roslyn/pull/37201#discussion_r303557804) [](ancestors = 303557804)